src/components/Visits/index.js

```diff
@@ -9,9 +10,9 @@ import DoctorsContext from "../../contexts/DoctorsContext";
 import VisitInputs from "../VisitInputs";
 import VisitItem from "../VisitItem";
 import validateVisit from "../../helpers/validateVisit";
-
 import "./visits.scss";
 
+let visitsInitial;
```

Comment:
what is that? you can't mutate in a reaction, especially not in a global variable